src/main/java/org/mskcc/cbio/portal/dao/DaoSampleProfile.java

```diff
@@ -160,14 +134,20 @@ public static boolean sampleExistsInGeneticProfile(int sampleId, int geneticProf
         try {
             con = JdbcUtil.getDbConnection(DaoSampleProfile.class);
             pstmt = con.prepareStatement
-                    ("SELECT * FROM sample_profile WHERE SAMPLE_ID = ? AND GENETIC_PROFILE_ID = ?");
+                    ("SELECT PANEL_ID FROM sample_profile WHERE SAMPLE_ID = ? AND GENETIC_PROFILE_ID = ?");
             pstmt.setInt(1, sampleId);
             pstmt.setInt(2, geneticProfileId);
             rs = pstmt.executeQuery();
-            return (rs.next());
```

Comment:
This seems like the function was broken before. Is this a bugfix? Or was this function previously only providing functionality of "sampleXInProfileYDoesHaveAPanelIdIdentified" (boolean)

Reply:
The function worked fine. It's not a bug fix. It's just removing the code that is not used anymore. Switching to `upsertSampleToProfileMapping` made this check not needed.

Reply:
👍